src/DataModelRegistry.hpp

```diff
@@ -50,36 +52,36 @@ class NODE_EDITOR_PUBLIC DataModelRegistry
 
   template<typename ModelType, bool TypeConverter = false>
   void
-  registerModel(std::unique_ptr<ModelType> uniqueModel = std::make_unique<ModelType>(), QString const &category = "Nodes")
+  registerModel(RegistryItemCloner cloner = [](){ return std::make_unique<ModelType>(); },
```

Comment:
`registerModel(RegistryItemCreator creator = `

Lambdas with no arguments can be defined without the `()`s, so you could just do `[] { return std::make_unique<ModelType>(); }`, but that honestly doesn't really matter